content/articles/convert-html-to-pdf-with-docraptor/index.md

```diff
@@ -0,0 +1,175 @@
+### Introduction
+It is easy for anyone to deliver PDFs because it is a standard forat for document presentation. Everything is given as a PDF now, including requests, receipts, and appropriation reports. However, the planned duration of PDF can be inconvenient if you need to create a large number of PDFs quickly.
+
+DocRaptor is an HTML-to-PDF API that greatly enhances the current situation in addition to having a PHP framework for simple joining. This article will guide the reader on how to use DocRaptor to converts HTML into a PDF.
+
+### Table of content
+- [Installation](#installation)
+- [Authentication](#authentication)
+- [Add HTML or a URL](#add-html-or-a-url)
+- [API options](#api-options)
+- [Get documentation](#get-documentation)
+- [Conclusion](#conclusion)
+  - [Further activity reading](#further-activity-reading)
+  
+### Installation
+Download the most recent version of DocRaptor to your computer and unzip it into your hard drive. You have two options for for doing this the first is using an author and the second is when working alone.
+
+If you are collaborating with an author, you will need to mention their name in the purchase line of your order confirmation.
+
+```javascript
+composer require docraptor/docraptor
+```
+
+- If you are not working with an author, you should download the most recent version and unzip it into your work area instead. Then, create a new record with the name `docraptor.php` and include the file `autoload.php` in it.
+
+```javascript
+require_once('./path/to/docraptor-php/autoload.php');
+```
+
+### Authentication
+You can use the API key `KEY YOUR API HERE`, which will not require you to have a record, but this will only work with watermarked archives and will not work with unwatermarked archives. In any case, you will need to have a track record that is up to date.
+
+```javascript
+$configuration=DocRaptor\configuration::getDefaultconfiguration();
+$configuration->setUsername('KEY_YOUR_API_HERE');
+```

```

Comment:
What are archives in this case? Would someone who has no programming knowledge know what you mean here?